src/tequila/grouping/overlapping_methods.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+from tequila.grouping.binary_utils import sorted_insertion_grouping, term_commutes_with_group
+from copy import deepcopy
+
+class OverlappingHelp:
+    '''
+    Class required for passing cov_dict and number of iterations to
+    OverlappingGroups. Eventually, this may also be used for building cov_dict from
+    approximate wavefunction if the user provides only the latter.
+    '''
+    def __init__(self, cov_dict, n_iter=5):
+        self.cov_dict = cov_dict
+        self.n_iter = n_iter
+
+class OverlappingGroups_wo_fixed:
+    '''
+    Class required for mapping OverlappingGroups onto a vector of overlapping coefficients.
+    Same as OverlappingGroups, but Pauli products corresponding to fixed coefficients are removed.
```

Comment:
Could you explain what fixed coefficients are? If it's difficult making a reference to something specific Aadi's paper would help.